internal/nonkube/client/fs/path_provider.go

```diff
@@ -0,0 +1,19 @@
+package fs
+
+import "fmt"
+
+type PathProvider struct {
+	Namespace string
+}
+
+func (p *PathProvider) getDefaultNamespace() string {
+	return ".local/share/skupper/default/prepared/state"
+}
+
+func (p *PathProvider) GetNamespace() string {
+
+	if p.Namespace == "" {
+		return p.getDefaultNamespace()
+	}
+	return fmt.Sprintf(".local/share/skupper/%s/prepared/state", p.Namespace)
```

Comment:
Same change here.
From: `.local/share/skupper/%s/prepared/state` to: `.local/share/skupper/%s/sources`

Reply:
fixed c97bcfd8fbe7bc28d8e903ac4cb1146e6b07cdf1